src/bakery.ts

```diff
@@ -12,17 +12,17 @@
  * ```
  */
 
-import macaroonlib, {
-  Macaroon,
-  Error as MacaroonError,
-  MacaroonObject,
-} from "macaroon";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
If I remove this ts-ignore I don't seem to get any errors. What was the issue here?

Reply:
Nice catch! this is a trace of my fight with TS :)